src/frontend/src/styles/components/management-tile.scss

```diff
@@ -44,9 +47,3 @@
         row-gap: $spacing-06;
     }
 }
-
-.management-tile:hover {
-    color: white;
-    background-color: var($sms-color-neutral-background-active);
-    transition: all 0.3s ease-out;
-}
```

Comment:
I grew tired of this for obvious reasons, fun to play around with but annyoing to keep